### DISCUSSION TITLE
Any font bigger than u8g2_font_logisoso62_tn font?

### DISCUSSION THREAD

**Discussion by Leo928** [olikraus/u8g2#2469] [Q&A]
u8g2_font_logisoso62_tn is a nice font but we would like something slightly bigger.  The next one **jump from 62 to 78** that turn out to be too big.  

Any chance of getting font larger than 62 but smaller than 78.   **Can we "make it"?**   

Any advise of **similar** high visibility number font (just 0 to 9) font(s) that can be read over long distance?

Many thanks

**Comment by olikraus** [comment]
Did you check the FAQ?
https://github.com/olikraus/u8g2/blob/dc9fc73ddfc4fc8690afea370780ba322bd0cd79/doc/faq.txt#L255-L288

**Comment by Leo928** [comment]
Many thanks for making the very nice software and supporting it!!!
